src/hooks/usePublicMessages.js

```diff
@@ -2,14 +2,16 @@ import {useCallback, useEffect, useMemo, useState} from 'react';
 
 /**
  * @typedef PublicMessageObject
- * @property {String} key - a unique key for a route/message pair.
+ * @property {Number} key - a unique key for the message, determined by the message ID from InfoPoint
```

Comment:
(usages bubble out so commenting on here) - I think we can just name this `messageId` now - the main reason I named this `key` before is because it was conditionally concatenating route/message ids to enforce uniqueness

Reply:
I'm gonna opt for just `id` since the scope under which you'll be accessing it will always be a `PublicMessageObject`

Reply:
Oh yes that would be most appropriate